File: 242-valid-anagram/242-valid-anagram.cs
public class Solution {
    public bool IsAnagram(string s, string t) {
        char[] sc = s.ToCharArray();
        char[] tc = t.ToCharArray();
        
        Array.Sort(sc);
        Array.Sort(tc);
        s = new string(sc);
        t = new string(tc);
        
        if(s == t)
            return true;
        
        
        return false;
        
    }
}

File: 242-valid-anagram/242-valid-anagram.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        int cnt[26]={0};
        for(char ch:s){
            cnt[ch-'a']++;
        }
        for(char ch:t){
            if(cnt[ch-'a']--<=0)
                return false;
        }
        for(int i:cnt){
            if(i)
                return false;
        }
        return true;
    }
};